css/base.css

```diff
@@ -1,10 +1,31 @@
 /* General Stuff */
 html, body {
   font-family: 'Roboto', sans-serif;
+  height: 100%;
   overflow-x: hidden;
   background-color: #212121;
 }
 
+#main-body-container {
+  min-height: 100%;
+  margin-bottom: -1.4em;
+  overflow: hidden;
+}
+
+#main-body-container::after {
+  content: "";
+  display: block;
+  height: 1.4em;
+}
+
+.navbar-nav > li, .navbar-nav > li > a {
+  transition: all 0.2s;
```

Comment:
This is causing some crazy purple jumpy shit on the nav bar. Only add transitions to the property you're changing (background).

Reply:
Done